### DISCUSSION TITLE
[Cameras] How to reset the camera position

### DISCUSSION THREAD

**Discussion by futahei** [google/model-viewer#3512] [Q&A] [answered]
### Description

I would like to reset the camera position with a function call.
In model-viewer, cameraOrbit must be set to change the camera position.
However, there is a problem regarding the camera distance. $radius in cameraOrbit, it is not working when zoomed in. To be precise, the camera will move away somewhat, but not to the intended position.

#### Live Demo

https://glitch.com/edit/#!/thin-regal-soldier

### Version

- recent commit ([8e52e80528f37d267e1717a89a2da89c71e52bf3](https://github.com/google/model-viewer/commit/8e52e80528f37d267e1717a89a2da89c71e52bf3))

### Browser Affected

- [x] Chrome, version: xx.x.xxxx.xx
- [x] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [x] Windows

**Comment by elalish** [comment, marked as answer]
Zoom uses a combination of radius and field of view change, so you'll need to reset both.
